src/software/ai/motion_constraint/motion_constraint_visitor.cpp

```diff
@@ -69,7 +69,11 @@ void MotionConstraintVisitor::visit(const ShootGoalTactic &tactic)
     });
 }
 
-void MotionConstraintVisitor::visit(const PasserTactic &tactic) {}
+void MotionConstraintVisitor::visit(const PasserTactic &tactic) {
+    current_allowed_constraints = std::set<MotionConstraint>({
+        MotionConstraint::HALF_METER_AROUND_BALL
+    });
```

Comment:
why this change? Why not use avoid_ball flag?

Reply:
removed it, think it was hack from earlier that slipped in